auslib/test/admin/views/test_permissions.py

```diff
@@ -250,17 +319,17 @@ def testGetScheduledChanges(self):
                 {
                     "sc_id": 1, "when": 10000000, "scheduled_by": "bill", "change_type": "insert", "complete": False, "sc_data_version": 1,
                     "permission": "rule", "username": "janet", "options": {"products": ["foo"]}, "data_version": None,
-                    "signoffs": {"bill": "releng"},
+                    "signoffs": {"bill": "releng"}, "required_signoffs": {},
                 },
                 {
                     "sc_id": 2, "when": 20000000, "scheduled_by": "bill", "change_type": "update", "complete": False, "sc_data_version": 1,
                     "permission": "release_locale", "username": "ashanti", "options": None, "data_version": 1,
-                    "signoffs": {"bill": "releng", "mary": "relman"},
+                    "signoffs": {"bill": "releng", "mary": "relman"}, "required_signoffs": {"releng": 1, "relman": 1},
```

Comment:
...and because they've got access to all products, we must take all required signoffs into consideration. I have a feeling we're going to want to change this a little bit in the future. Eg: if end up with a "seamonkey" role, and I don't think we should require their signoff to a a new full fledged admin user.